test/e2e/e2e-test/docker/go/Dockerfile.go

```diff
@@ -13,16 +13,24 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM golang:1.12
+FROM golang:1.12 AS builder
+
+ARG COMMIT_HASH=fdb185d66faddad1651c18150d63bae32610f3ac
+ARG GO2SKY_REPO=${COMMIT_HASH}.tar.gz
```

Comment:
`${COMMIT_HASH}.tar.gz` is not `GO2SKY_REPO`, `https://github.com/SkyAPM/go2sky/archive/${GO2SKY_REPO}` is